avocado/core/varianter.py

```diff
@@ -467,6 +467,15 @@ def to_str(self, summary=0, variants=0, **kwargs):
         :param kwargs: Other free-form arguments
         :rtype: str
         """
+        if self._no_variants == 0:  # No variants, only defaults:
+            out = []
+            if summary:
+                out.append("No variants available, using defaults only")
+            if variants:
+                out.append("\n".join(variant_to_str(next(self.itertests()),
+                                                    variants - 1, kwargs,
+                                                    self.debug)))
+            return "\n\n".join(out)
```

Comment:
I thought whether we should fake the name with `Variant :` or keep it None as it might help advanced users to understand whether variant was actually generated (0 vs. no variants) so I went with this as it is what is actually supplied to test. Anyway I'm open to using `""` as well.